### ISSUE TITLE
Background Blur is not working

### ISSUE THREAD

**Issue by yangsibai** [L2jLiga/xd2svg#33]
## Expected Behavior

Background Blur is working.

![image](https://user-images.githubusercontent.com/2907211/45408731-cf5f8880-b69f-11e8-8a7b-cf6f9a74254f.png)

## Actual Behavior

Not working.

![image](https://user-images.githubusercontent.com/2907211/45408725-ca023e00-b69f-11e8-918a-fd4d36830ad5.png)

## Steps to Reproduce the Problem

Add a shape, check the background blur, the exported svg don't have blur.

## Specifications

  - Version: 1.0.0-2
  - Platform: macOS

**Comment by L2jLiga** [comment]
Root of this problem is that when you create background blur actually you create the shape over your shape with background. Looks like this things work different in XD and SVG.

Currently I have no ideas how to solve that, @yangsibai have you any ideas?

[minimal example](http://jsfiddle.net/5x2wde1z/10)

**Comment by yangsibai** [comment]
It can be implemented by combine filters.

For example:

```
<filter id="blurMe" x="0" y="0" width="100%" height="100%">
  <feGaussianBlur in="SourceGraphic" stdDeviation="15" result="blur"/>
  <feComponentTransfer in="blur" result="op">
    <feFuncR type="linear" slope="50"/>
    <feFuncG type="linear" slope="50"/>
    <feFuncB type="linear" slope="50"/>
    <feFuncA type="linear" slope="0.56"></feFuncA>
  </feComponentTransfer>
</filter>
<image xlink:href="./image1.png" width="338" height="338" />
<rect x="50" y="100" width="200" height="60" fill="white" filter="url(#blurMe)" />
```

After some tests, I guess:

+ `stdDeviation` of `feGaussianBlur` is the blur amount.
+ `slope` of `feFuncR`, `feFuncG`, `feFuncB` can be used for brightness. (In XD, the value is -50~50, I think it need some transform to apply the value)
+ `slope` of `feFuncA` can be used for opacity.

References:

+ <https://developer.mozilla.org/en-US/docs/Web/SVG/Element/feComponentTransfer>
+ <https://webplatform.github.io/docs/svg/elements/feFuncA/>
+ <https://webplatform.github.io/docs/css/functions/brightness/>

@L2jLiga

**Comment by yangsibai** [comment]
![image](https://user-images.githubusercontent.com/2907211/45729100-7e024c80-bbfd-11e8-989b-88814c5ae575.png)

[blur-svg.zip](https://github.com/L2jLiga/xd2svg/files/2395281/blur-svg.zip)

**Comment by L2jLiga** [comment]
Good idea to use `feComponentTransfer` and I guess it will be used by xd2svg in future, but it is exactly not what we are looking for..

About your example: we just have rectangle over the image with white half-transparent backgrounds, image still not blurred.


@yangsibai

**Comment by yangsibai** [comment]
Image is blurred by `feGaussianBlur`, it's not just half-transparent, it's blurred. I haven't find out how to map the parameters, but it's work.

@L2jLiga

**Comment by L2jLiga** [comment]
@yangsibai I tried to use your solution and find something too..
Currently I decied to release 1.0.0 version, at least it can do more then Adobe XD when you try to export to svg :)

Best :)
